app/src/main/java/com/konkuk/medicarecall/ui/feature/home/model/ImmediateCallRequestDto.kt

```diff
@@ -1,4 +1,4 @@
-package com.konkuk.medicarecall.ui.home.model
+package com.konkuk.medicarecall.ui.feature.home.model
```

Comment:
여기도 데이터 패키지 안의 request 패키지에 넣는 건 어떨까요 ??
```suggestion
package com.konkuk.medicarecall.data.dto.request
```

Reply:
넵 해당 부분은 월요일 회의에서 논의한 후 진행하기로해서 현상 유지했습니다!